### PR TITLE
Add groundcover.com to ebpf.io case studies

### PR DESCRIPTION
**Pull request by shaharazulay**
Adding groundocover.com, which uses eBPF to reinvent the cloud-native application monitoring domain, to the ebpf.io case studies.

### CODE REVIEW

**Comment by xmulligan** [comment]
Could you make the description a bit more about the technology and the open source projects that you have?

**Comment by shaharazulay** [comment]
@xmulligan changed the description to fit the format of the other descriptions.

File: src/data/shared/case-studies/case-studies-list.js [hunk 1]
```diff
@@ -494,7 +494,7 @@ const caseStudiesData = {
     {
       title: 'groundcover',
       logo: groundcover,
-      description: '<strong>groundcover</strong> uses eBPF to reinvent the cloud-native application monitoring domain. It enables teams to instantly monitor everything they build and run in the cloud without compromising on cost, granularity, or scale.',
+      description: '<strong>groundcover</strong> uses eBPF to instantly monitor application performance in cloud-native environments, at any scale.',
```
Comment (xmulligan) [inline src/data/shared/case-studies/case-studies-list.js:497]:
```suggestion
      description: '<strong>groundcover</strong> uses eBPF to instantly monitor application performance in cloud native environments, at any scale.',
```

File: src/data/shared/case-studies/case-studies-list.js [hunk 2]
```diff
@@ -1589,7 +1589,7 @@ const caseStudiesData = {
...
```
Comment (xmulligan) [inline src/data/shared/case-studies/case-studies-list.js:1592]:
```suggestion
      description: '<strong>groundcover</strong> uses eBPF to instantly monitor application performance in cloud native environments, at any scale.',
```

File: src/data/shared/case-studies/case-studies-list.js [hunk 3]
```diff
@@ -2020,7 +2020,7 @@ const caseStudiesData = {
...
```
Comment (xmulligan) [inline src/data/shared/case-studies/case-studies-list.js:2023]:
```suggestion
      description: '<strong>groundcover</strong> uses eBPF to instantly monitor application performance in cloud native environments, at any scale.',
```

**Review by xmulligan — APPROVED** [review]

**Pull request merged**